src/modules/membership-builder/MembershipBuilderPage.vue

```diff
@@ -5,7 +5,11 @@
         <font-awesome-icon :icon="['fa', 'check-circle']" />
         {{ t('form.updated') }}
       </span>
-      <span v-if="dirty" class="mr-4">
+      <span v-else-if="validation.$invalid" class="mr-4 text-danger">
+        <font-awesome-icon :icon="['fa', 'warning']" />
+        {{ t('form.errors.aggregator') }}
```

Comment:
Bit confusing that it says "check the fields above", we should add another string which is "check for errors below" or something